ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/InternalCustomerCenter.kt

```diff
@@ -120,6 +169,143 @@ private fun CustomerCenterLoaded(
     }
 }
 
+@Composable
+private fun RestorePurchasesDialog(
+    state: RestorePurchasesState,
+    onDismiss: () -> Unit,
+    onRestore: () -> Unit,
+    onContactSupport: () -> Unit,
+) {
+    var isRestoring by remember { mutableStateOf(false) }
+
+    // Reset isRestoring when dialog is dismissed
```

Comment:
Hmm interesting... I thought, once the dialog is dismissed (and the composable removed from the tree), the state should also be lost and reset... Is this needed?

Reply:
No, you're correct. Removing